paper.md

```diff
@@ -22,48 +22,47 @@ bibliography: paper.bib
+See the [homepage](https://github.com/Beuth-Erdelt/DBMS-Benchmarker) and the [documentation](https://dbmsbenchmarker.readthedocs.io/en/latest/Docs.html) for more details.
 
 # Statement of Need
 
 Performance benchmarking of database management systems (DBMS) is an active research area and has a broad audience. It is used "*by DBMS developers to evaluate their work and to find out which algorithm works best in which situation. Benchmarks are used by (potential) customers to evaluate what system or hardware to buy or rent. Benchmarks are used by administrators to find bottlenecks and adjust configurations. Benchmarks are used by users to compare semantically equivalent queries and to find the best formulation alternative*", @10.1007/978-3-030-84924-5_6.
-Also in the academic field, approaches and their special implementations are examined in benchmarks.
+Approaches and their special implementations are also examined in benchmarks in academia.
 There is a large variety of DBMS types and products.
-For example @DBEngines ranks 350 DBMS (150 Relational), @DBDBIO lists 850 DBMS (280 Relational).
+For example, @DBEngines ranks 350 DBMS (150 Relational) and @DBDBIO lists 850 DBMS (280 Relational).
 We focus on Relational DBMS (RDBMS) in the following.
-The types thereof can be divided into for example row-wise, column-wise, in-memory, distributed and GPU-enhanced. 
+Their types can be divided into, for example, row-wise, column-wise, in-memory, distributed, and GPU-enhanced. 
 All of these products have unique characteristics, special use cases, advantages and disadvantages, and their own justification.
-In order to be able to verify and ensure the performance measurement, we want to be able to create and repeat benchmarking scenarios.
-Repetition and thorough evaluation are crucial, in particular in the age of Cloud-based systems with its diversity of hardware configurations, @Raasveldt2018FBC32099503209955, @DBLPconfsigmodKerstenKZ18, @KounevLK20.
+In order to be able to verify and ensure performance measurements, we want to be able to create and repeat benchmarking scenarios.
+Repetition and thorough evaluation are crucial, in particular in the age of cloud-based systems with their diversity of hardware configurations[@Raasveldt2018FBC32099503209955; @DBLPconfsigmodKerstenKZ18; @KounevLK20].
 
-Thus there is a widespread need for a tool to support the repetition and reproducibility of benchmarking situations, and that is capable of connecting to all these systems.
+Thus there is widespread need for a tool to support the repetition and reproducibility of benchmarking situations, and that is capable of connecting to all these systems.
 
-When we collect a lot of data during benchmarking processes, we also need a tool that will help with the statistical, visual and interactive analysis of the results.
-The authors advocate using Python as a common Data Science language, since
```

Comment:
I think Data Science is more appropriate

Reply:
Ok, I can change it back if you confirm that's what you want. I personally think that this is in the process of changing, much like Cloud changed to cloud. Caps are really for proper names, and I don't think Data Science is a proper name anymore, more a descriptive term.  And for terms like this "data science" is how they are used as a noun, but "data-science" is the way they are used as adjectives, like "data-science methods"

Reply:
Yes, please change that one thing back. I understand that the English language suggests it the other way and I think it will eventually change that way, but at the moment unhyphenated is much more common.
Thank you for helping to improve the paper, I appreciate it!

Reply:
ok, now that change has been undone